airflow/api_fastapi/execution_api/datamodels/taskinstance.py

```diff
@@ -61,6 +61,30 @@ class TITargetStatePayload(BaseModel):
     state: IntermediateTIState
 
 
+class TIDeferredStatePayload(BaseModel):
+    """Schema for updating TaskInstance to a deferred state."""
+
+    state: Annotated[
+        Literal[IntermediateTIState.DEFERRED],
+        # Specify a default in the schema, but not in code, so Pydantic marks it as required.
+        WithJsonSchema(
+            {
+                "type": "string",
+                "enum": [IntermediateTIState.DEFERRED],
+                "default": IntermediateTIState.DEFERRED,
+            }
+        ),
+    ]
+
+    classpath: str
+    kwargs: dict[str, Any]
+    created_date: UtcDateTime
+    next_method: str
+    # need to serialise to datetime.timedelta
+    timeout: str | None
+    # what about triggerer_id?
```

Comment:
This gets set in the db once the API processes the request, so it shouldn't/can't be part of the request

Reply:
The trigger_id right? I need to remove this comment then

Reply:
Yeah

Reply:
Yeah, removing the comment